### DISCUSSION TITLE
Scope

### DISCUSSION THREAD

**Discussion by lishaduck** [PSDTools/app#17] [Polls]
> Formerly: Feasibility of the App
> See #13

So, I was thinking about the scope of our project and realized maybe we ought to think about something—we can't do everything. We want to, but it does not seem feasible to me. Anyway, that is not what the ELO is even about. It's about [Pirate Code 3.0](https://github.com/PSDTools/pirate-code-3.0).
I think it is fine to want lots of features for the app, even to plan for them, but without broader support and more people, I think that it is completely unreasonable to dive into building all of this in six months. The previous Pattonville app was 7 people in 12 months, and we aren't in their situation. We need to step back and think "What are we even doing?"
I am not trying to say that it is impossible, but it seems infeasible. We need to consider what we won't do, and what we will. What we want to, and what we don't. We don't want to get in over our heads.

**CC:** @ParkerH27

**Comment by PetalCat** [comment]
I think we should keep this in mind while developing because the main goal of this app is. after all. the wallet, and only secondly is it to wrap all sorts of extra stuff into it. The wallet portion has more priority because that's what the ELO is and that is what I have pitched to the admin.

**Reply by lishaduck** [reply]
Yeah. I agree (as you can tell). 👍
